src/compat.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Copyright (C) 2019-2020 Uwe Bonnes <bon@elektron.ikp.physik.tu-darmstadt.de>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.	 If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* Compatibility functions wrapping compiler-specific code, especially
```

Comment:
The code base comment style for multi-line comments is
```c
/*
 * comment
 * lines
 */
```

Reply:
Fixed. While at it I also fixed the directly preceding comment in `cortexm.c`.

Reply:
Oops, didn't notice the leading blank line, only the trailing one. Fixed now. Thanks for your patience!

Reply:
No worries, thanks for yours!